app/models/hackbot/interactions/sql.rb

```diff
@@ -0,0 +1,133 @@
+class Executer
+  def execute(query)
```

Comment:
There's a ton of personally identifiable information in our database which I don't feel good about not putting any measures in place to protect.

For leaders specifically, I'm worried about:

- Names (could we just render someones full name?)
- Phone number
- Addresses (and lat/long)
- Email
- Usernames
- Slack ID